build.cmd

```diff
@@ -26,6 +26,7 @@ set __SkipTestBuild=
 if "%1" == "" goto ArgsDone
 if /i "%1" == "/?" goto Usage
 if /i "%1" == "x64"    (set __BuildArch=x64&&shift&goto Arg_Loop)
+if /i "%1" == "x86"    (set __BuildArch=x86&&shift&goto Arg_Loop)
```

Comment:
@jasonwilliams200OK Thanks but I already modified the Unix build scripts for x86. Unfortunately it looks like there's a lot of code that needs to be fixed/ported for a Unix x86 version to build successfully and I haven't had time for that kind of work lately. Anyway, I'm more interested in getting the JIT to work on x86 (win) but that's difficult without the JIT team's cooperation.